src/_help/enhance-documentation-content/markdown-support.md

```diff
@@ -133,9 +137,11 @@ You can use any of the following CSS length units as `unit`:
 
 ## Markdown files as an external reference
 
-Markdown files can be included as an [external reference](/help/specification-support/references) within your contract document with the $ref syntax `$ref: "./path/to/local-markdown.md"`. In the same way you can extract part of your contract (usually JSON schema of your models into dedicated `*.yaml` or `*.json` files), you can extract your markdown content into dedicated files too.
+Markdown files can be included as an [external reference](/help/specification-support/references) within your API document with the $ref syntax `$ref: "./path/to/local-markdown.md"`.
```

Comment:
```suggestion
Markdown files can be included as an [external reference](/help/specification-support/references) within your API definition with the $ref syntax `$ref: "./path/to/local-markdown.md"`.
```

Reply:
Both work here, but reading your comments Chris, I think that it does make sense to only refer to the "document" word when talking about the file. Here, we talke about the spec usage, so the definition word seems to work better 👍